src/utils/types.ts

```diff
@@ -1,10 +1,22 @@
 export interface AlumniProfileInfo {
-    name: string;
-    program: string;
-    batch: string;
-    image?: { url: string };
-    slogan?: string;
-    tags: string[];
-    history: { year: string; position: string; company: string }[];
-    slug: string;
-  }
+  name: string
+  program: string
+  batch: string
+  image?: { url: string }
+  slogan?: string
+  tags: string[]
+  history: { year: string; position: string; company: string }[]
+  slug: string
+}
+
+export interface EventData {
+  id: string
+  Title: string
```

Comment:
I would always suggest following an uniform naming convention that languages suggests. At least uniform within the project. (_**camelCase** in our case_)

@axyut Whats your though on this?